artifacts/fedora/fedora_test.go

```diff
@@ -16,7 +16,7 @@ import (
 var _ = Describe("Fedora", func() {
 	DescribeTable("Inspect should be able to parse releases files",
 		func(release, mockFile string, details *api.ArtifactDetails, envVariables map[string]string, metadata *api.Metadata) {
-			c := New(release, envVariables)
+			c := New(release, "x86_64", envVariables)
```

Comment:
please change this to constant

Reply:
We don't have constants of other values, why this one should be?